google/google-how-to/gce/check-cve-on-instance.rst

```diff
@@ -0,0 +1,88 @@
+
+We need to download and merge the OVAL feeds of the repositories configured on the instance before running a scan.
+
+Install :code:`oval-xml-feed-merge` on the instance to merge OVAL feeds.
+
+.. code::
+
+    sudo snap install oval-xml-feed-merge
+
+Download the OVAL feeds. The merge tool can access only the home directory so please download the OVAL feeds there. The merge tool can take upto a few minutes if you are running an instance with a conservative CPU configuration.
+
+.. code::
+
+    wget "https://security-metadata.canonical.com/oval/com.ubuntu.esm-apps_$(lsb_release -cs).cve.oval.xml.bz2"
+    bunzip2 "com.ubuntu.esm-apps_$(lsb_release -cs).cve.oval.xml.bz2"
+    wget "https://security-metadata.canonical.com/oval/com.ubuntu.$(lsb_release -cs).cve.oval.xml.bz2"
+    bunzip2 "com.ubuntu.$(lsb_release -cs).cve.oval.xml.bz2"
+    oval-xml-feed-merge --verbose "com.ubuntu.$(lsb_release -cs).cve.oval.xml" "com.ubuntu.esm-apps_$(lsb_release -cs).cve.oval.xml" --output "com.ubuntu.$(lsb_release -cs)-pro.cve.oval.xml" # The order of filenames in this command is important
+
+If you are running a release older than five years, run the following commands instead to download and merge the OVAL feeds.
+
+.. code::
+
+    wget "https://security-metadata.canonical.com/oval/com.ubuntu.esm-apps_$(lsb_release -cs).cve.oval.xml.bz2"
+    bunzip2 "com.ubuntu.esm-apps_$(lsb_release -cs).cve.oval.xml.bz2"
+    wget "https://security-metadata.canonical.com/oval/com.ubuntu.esm-infra_$(lsb_release -cs).cve.oval.xml.bz2"
+    bunzip2 "com.ubuntu.esm-infra_$(lsb_release -cs).cve.oval.xml.bz2"
+    wget "https://security-metadata.canonical.com/oval/com.ubuntu.$(lsb_release -cs).cve.oval.xml.bz2"
+    bunzip2 "com.ubuntu.$(lsb_release -cs).cve.oval.xml.bz2"
+    oval-xml-feed-merge --verbose "com.ubuntu.$(lsb_release -cs).cve.oval.xml" "com.ubuntu.esm-apps_$(lsb_release -cs).cve.oval.xml" "com.ubuntu.esm-apps_$(lsb_release -cs).cve.oval.xml" --output "com.ubuntu.$(lsb_release -cs)-pro.cve.oval.xml" # The order of filenames in this command is important
+
+Finally, run the scan,
+
+.. code::
+
+    oscap oval eval --report report.html "com.ubuntu.$(lsb_release -cs)-pro.cve.oval.xml"
+
+Downloading the scan report
+---------------------------
+
+Now you can download the report from the instance to your local computer using the gcloud CLI.
+The following command assumes you ran the :code:`oval eval` in the home directory of a user named "ubuntu_user" and that the name of the instance is "ubuntu-instance".
+
+.. code::
+
+    gcloud compute scp ubuntu_user@ubuntu-instance:report.html .
+
+Now you have the "report.html" in the directory where you executed the :code:`gcloud compute scp` command. If you open the report in a browser you will see something like the screenshot below. You will be able to tell by color code of the CVE (Reference ID column) in the results table if the CVE is addressed on your instance in it's current state.
+
```

Comment:
Could we add a statement here mentioning that green indicates an addressed CVE, while orange indicates otherwise? The other colours need not be explained, but if there is some documentation about OVAL tooling where that is explained, it might be a good idea to point to it.

Reply:
Added information about the colors but couldn't find a relevant documentation link.